sdk-extensions/autoconfigure/src/testOtlp/java/io/opentelemetry/sdk/autoconfigure/OtlpGrpcConfigTest.java

```diff
@@ -324,4 +327,29 @@ void configuresGlobal() {
               assertThat(server.metricRequests).isNotEmpty();
             });
   }
+
+  static void shutdownGlobalSdk() {
```

Comment:
I merged the `testOtlpGrpc` and `testOtlpHttp` test sets, but ran into some problems because both test suites had a test that sets up `GlobalOpenTelemetry` with a periodic metric reader that continues to export after the test is over and the test grpc / http server is shutdown.

The solution I came up with is to use reflection to obtain `OpenTelemetrySdk` from `GlobalOpenTelemetry` and call shutdown. 

We might consider adding a utility to this to the `sdk-testing` in case users need to set up `GlobalOpenTelemetry` for some reason, and need to cleanup after.